.evergreen.yml

```diff
@@ -1,8 +1,30 @@
 buildvariants:
-- display_name: Ubuntu 18.04
-  name: ubuntu1804
+- display_name: Python 3.9
+  name: cp39
   run_on:
-  - ubuntu1804-small
+  - ubuntu2404-small
+  expansions:
+    python_toolchain: /opt/python/3.9
+  tasks:
+  - name: unit_tests
+  - name: evg_api_install
+
+- display_name: Python 3.10
+  name: cp310
+  run_on:
+  - ubuntu2404-small
+  expansions:
+    python_toolchain: /opt/python/3.10
+  tasks:
+  - name: unit_tests
+  - name: evg_api_install
```

Comment:
Modernize base image and test back to oldest supported version in our pyproject.toml